addons/website_event_booth_sale/models/sale_order.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import Command
+from odoo import models, _
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    def _cart_find_product_line(self, product_id=None, line_id=None, **kwargs):
```

Comment:
As this is called from ``_cart_update`` that defers its ``kwargs`` to ``_cart_find_product_line`` can't we somehow have parameters coming from kwargs instead of relying on context ?